src/expr/src/scalar/func.rs

```diff
@@ -2460,6 +2504,22 @@ impl BinaryFunc {
             }
             BinaryFunc::ToCharTimestamp => Ok(eager!(to_char_timestamp, temp_storage)),
             BinaryFunc::ToCharTimestampTz => Ok(eager!(to_char_timestamptz, temp_storage)),
+            BinaryFunc::WindowTumblingTs => {
+                eager!(|a: Datum, b: Datum| window_tumbling(
+                    a.unwrap_timestamp(),
+                    b.unwrap_interval(),
+                    NaiveDateTime::from_timestamp(0, 0),
```

Comment:
We should document that this is considered from the origin (and we might want to eventually expose a variant of this function that allows a custom origin, similar to `date_bin`).

Reply:
This idiom is used in a number of places in the code and the underlying library is the one that enforces that this is from the Unix epoch, so I don't think documenting it in the code here's necessary.

And yep--an obvious extension to let folks control what the origin is, but don't think ti's imperative to add it until e.g. there's some request for it.

Reply:
Ah, sorry, I meant to say document in the user docs, not in the code! As otherwise you might get surprising behavior if you use an interval that doesn't cleanly divide a day.

Reply:
Ah, I have a mention of that [here](https://github.com/MaterializeInc/materialize/pull/9453/files#diff-b9b2583fe620dc7ffd45d79d7319e5a39d1f0501dfde71615846ee697a350110R37), hence my confusion at your comment. I can make sure that's more apparent.